modules/lang-mustache/src/test/java/org/elasticsearch/script/mustache/SearchTemplateRequestXContentTests.java

```diff
@@ -125,41 +128,51 @@ public void testToXContentWithStoredTemplate() throws IOException {
     }
 
     public void testFromXContentWithEmbeddedTemplate() throws Exception {
-        String source = "{"
-            + "    'source' : {\n"
-            + "    'query': {\n"
-            + "      'terms': {\n"
-            + "        'status': [\n"
-            + "          '{{#status}}',\n"
-            + "          '{{.}}',\n"
-            + "          '{{/status}}'\n"
-            + "        ]\n"
-            + "      }\n"
-            + "    }\n"
-            + "  }"
-            + "}";
+        String source = """
+            {
+              'source' : {
```

Comment:
What do you think - is it worth removing the `s.replace("'", "\"")` in `newParser()` and using standard double quotes in the JSON, here and. below?